lib/src/r_tree/quickselect.dart

```diff
@@ -3,9 +3,8 @@ part of r_tree;
 
 // sort an array so that items come in groups of n unsorted items, with groups sorted between each other;
 // combines selection algorithm with binary divide & conquer approach
-multiSelect<E>(List<E> arr, int left, int right, int n, num Function(E) getter) {
+multiSelect<E>(List<E> arr, int left, int right, int n, int Function(E a, E b) compare) {
```

Comment:
#nit
```suggestion
void multiSelect<E>(List<E> arr, int left, int right, int n, int Function(E a, E b) compare) {
```

Reply:
we should probably add some stronger analysis options in a future PR

Reply:
#allthelints same with datatables